test/test_nn.py

```diff
@@ -12685,6 +12685,7 @@ def test_per_sample_weights(mode, trainable_scale):
         for mode, trainable in itertools.product(modes, trainable_scale):
             test_per_sample_weights(mode, trainable)
 
+    @unittest.skipIf(IS_WINDOWS, "FIXME: CUDA error: too many resources requested on 11.2")
```

Comment:
Could we also have a issue for this one, and reference the issue number here?

Reply:
https://github.com/pytorch/pytorch/issues/52002 yes I made one here (linked in description)